Header Andrii

Kirieiev opened the pull request.

Kirieiev committed:
first commit

NetworkVisus commented on src/partials/header.html:
```diff
@@ -1,4 +1,43 @@
+
+      <div class="header__logo" tabindex="0">
+        <a class="header__logo_link">
+          <div class="header__logo_container">
```
На планшеті лого знаходиться не на тому ж місці, де і на макеті
![image](https://github.com/NetworkVisus/emmet-group/assets/59713067/eae77601-ef4f-410b-8ba3-ebfd54b4b54a)
![image](https://github.com/NetworkVisus/emmet-group/assets/59713067/e06dce1b-f4e9-4a0d-a6de-cb86aefe8da1)

NetworkVisus commented on src/partials/header.html:
```diff
@@ -1,4 +1,43 @@
-  <h2>Header example commit</h2>
-</header>
+<div class="header__wrapper">
+  <header class="header">
```
header повинен огортати div, а не навпаки

NetworkVisus commented on src/partials/header.html:
```diff
@@ -1,4 +1,43 @@
+
+        <div class="header__icon_container">
+          <button type="button" class="header__icon_button">
+            <svg class="header__icon">
```
Для кнопки додай data-modal-open

NetworkVisus replied:
Хоча напевно це корзина, не потрібно модалку відкривати, не додавай

NetworkVisus reviewed (changes requested):
1. Подивись коменти по коду
2. letter-spacing в нас розраховується по принципу:

letter-spacing(в px)/font-size = letter-spacing (в em)

Поправ усюди, бо нам потрібно щоб letter-spacing був у em, а не px (!!!не забудь знак мінус, якщо він був, та не забудь замість px написати em після розрахунків вище)

Поки це поправ, далі будемо дивитись

Kirieiev committed:
second commit

NetworkVisus reviewed (changes requested):
1. Перейменуй всі класи, використовуй лише одне - замість __
2. В своему контейнері прибери падінги, додай окрім свого котейнера дефолтний .container (його не змінюємо взагалі, там вже все порописано)
3. Лого на десктопі тепер не стоїть по середині
![image](https://github.com/NetworkVisus/emmet-group/assets/59713067/9b755f30-2fca-4bbe-b91f-ae19b0d1c01f)

4. Айдішники якорів можеш взяти як в футері

Kirieiev committed:
Merge branch 'main' into Header-Andrii

Kirieiev committed:
third commit

Kirieiev committed:
fourth-commit

NetworkVisus committed:
bugfix/header

Deleted property changing in the default .container to avoid conflicts with other sections

NetworkVisus reviewed (approved):
.container не треба було міняти падінги - це може погано обернутися для інших секцій - я вже сам це виправив
